.travis.yml

```diff
@@ -16,7 +16,9 @@ matrix:
   include:
```

Comment:
exclude `python: 3.4` --> `python: 3.5" since we've changed the default version matrix to be 2.7 and 3.5